definitions/npm/axios_v0.19.x/flow_v0.25.x-v0.74.x/axios_v0.19.x.js

```diff
@@ -0,0 +1,178 @@
+declare module 'axios' {
+  declare interface AxiosTransformer<T> {
+    (data: T, headers?: Object): Object;
+  }
+  declare interface ProxyConfig {
+    host: string;
+    port: number;
+    auth?: {
+      username: string,
+      password: string,
+    };
+    protocol?: string;
+  }
+  declare interface Cancel {
+    constructor(message?: string): Cancel;
+    message: string;
+  }
+  declare interface Canceler {
+    (message?: string): void;
+  }
+  declare interface CancelTokenSource {
```

Comment:
Please switch to using classes or types, because interface often causes slowdowns for flow.

Reply:
Done